src/js_realm.hpp

```diff
@@ -759,14 +759,20 @@ void RealmClass<T>::delete_file(ContextType ctx, ObjectType this_object, Argumen
         throw std::runtime_error("Invalid argument, expected a Realm configuration object");
     }
 
-    ObjectType object = Value::validated_to_object(ctx, value);
+    ObjectType config_object = Value::validated_to_object(ctx, value);
     realm::Realm::Config config;
 
     static const String path_string = "path";
-    ValueType path_value = Object::get_property(ctx, object, path_string);
+    ValueType path_value = Object::get_property(ctx, config_object, path_string);
```

Comment:
Can it be moved after line 771 and before 772?

Reply:
Ugh 😑 It can, but will look ugly